CHANGELOG.md

```diff
@@ -8,15 +8,15 @@
 - `migrations`: when upgrading the schema version an explicit version must be
   provided. This is required to ensure at most one delivery.
 - `cmd/bnscli` can create a schema upgrade transaction.
-- Blue Account functionality was implememented in  `cmd/bnsd/x/blueaccount` and
+- Blue Account functionality was implememented in `cmd/bnsd/x/blueaccount` and
   integrated into `cmd/bnsd` application.
-  
+
 ## 0.21.3
 
 - `x/msgfee` was extended to provide a message to set a fee for a given message path.
-`bnscli` and `bnsd` were extended to support this change.
+  `bnscli` and `bnsd` were extended to support this change.
 
-Breaking changes
+### Breaking Changes
```

Comment:
I believe _Breakinch changes_ is not a header on purpose. This is similar to how [IOV core changelog](https://github.com/iov-one/iov-core/blob/master/CHANGELOG.md) looks like.

Reply:
I'll revert it back